Client/DeleteApi.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using InfluxDB.Client.Api.Domain;
+using InfluxDB.Client.Api.Service;
+using InfluxDB.Client.Core;
+using NodaTime;
+
+namespace InfluxDB.Client
+{
+    public class DeleteApi
+    {
+        private readonly DefaultService _service;
+
+        protected internal DeleteApi(DefaultService service)
+        {
+            Arguments.CheckNotNull(service, nameof(service));
+
+            _service = service;
+        }
+        
+        /// <summary>
+        /// Delete Time series data from InfluxDB.
+        /// </summary>
+        /// <param name="start">Start time</param>
+        /// <param name="stop">Stop time</param>
+        /// <param name="predicate">Sql where like delete statement</param>
+        /// <param name="bucket">The bucket from which data will be deleted</param>
+        /// <param name="org">The organization name of the above bucket</param>
```

Comment:
`The organization name`: the name should be remove because `org` could be `name` or `id`